config/jobs/kubernetes-incubator/service-catalog/service-catalog-presubmits.yaml

```diff
@@ -18,3 +18,28 @@ presubmits:
         env:
         - name: NO_DOCKER
           value: "1"
+  - name: pull-service-catalog-xbuild
+    agent: kubernetes
+    context: pull-service-catalog-xbuild
+    always_run: true
+    skip_report: false
+    rerun_command: "/test pull-service-catalog-xbuild"
+    trigger: "(?m)^/test (all|pull-service-catalog-xbuild)\\s*"
+    spec:
+      containers:
+      - args:
+        - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src/github.com/kubernetes-incubator/$(REPO_NAME)"
+        - "--timeout=90"
```

Comment:
does this take 90 minutes? if so, does it use many cores / gbs of ram? if it does, we should add a resource request. if it doesn't do any of those things, we should tighten the timeout.

Reply:
it's a go build, so it'll use cores we give it. What's the default? 1/2/500millicore?

the two make targets together take about 50minutes on travis, but much less locally, so I'm not sure how to gauge. 

This is pretty much the most cpu intensive workload we have. Others take long, but are filled with pauses and inefficiencies. This should be straight compiling.

Reply:
I'll change it to 60, if that sounds okay. What is the default?

Reply:
default is infinite I think